### PR TITLE
Make translators annotation based.

### PR DESCRIPTION
**Pull request by e3ndr**
Also renamed TranslatorsInit to Translators, since that makes more sense.

usage
```java
@Translator(packet = SomePacket.class)
public class SomeTranslator extends PacketTranslator<SomePacket> {
    // ...
}
```

### CODE REVIEW

File: bootstrap/standalone/src/main/java/org/geysermc/platform/standalone/Util.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package org.geysermc.platform.standalone;
+
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.OpenOption;
+import java.nio.file.Paths;
+
+import org.geysermc.platform.standalone.console.GeyserLogger;
+
+public class Util {
```
Comment (RednedEpic) [inline bootstrap/standalone/src/main/java/org/geysermc/platform/standalone/Util.java]: Naming is very vague here, maybe something like `LoopbackUtil` or `LoopbackUtils`.

File: bootstrap/standalone/src/main/java/org/geysermc/platform/standalone/Util.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                    }
+                }
+
+                String result = sb.toString();
+
+                if (!result.contains("minecraftuwp")) {
+                    Files.write(Paths.get(System.getenv("temp") + "/loopback_minecraft.bat"), loopbackCommand.getBytes(), new OpenOption[0]);
+                    process = Runtime.getRuntime().exec(startScript);
+
+                    geyserLogger.warning("Added loopback exemption to Windows!");
```
Comment (RednedEpic) [inline bootstrap/standalone/src/main/java/org/geysermc/platform/standalone/Util.java]: Shouldn't be a warning if the exemption was successfully applied.

File: connector/src/main/java/org/geysermc/connector/network/translators/Translators.java [hunk 3]
```diff
@@ -162,8 +141,36 @@ public static void start() {
         Registry.registerBedrock(InteractPacket.class, new BedrockInteractTranslator());
         Registry.registerBedrock(TextPacket.class, new BedrockTextTranslator());
         Registry.registerBedrock(RespawnPacket.class, new BedrockRespawnTranslator());
-        Registry.registerBedrock(ShowCreditsPacket.class, new BedrockShowCreditsTranslator());
-
+        Registry.registerBedrock(ShowCreditsPacket.class, new BedrockShowCreditsTranslator());*/
```
[the remark this answers is not in the document]
Comment (e3ndr) [reply connector/src/main/java/org/geysermc/connector/network/translators/Translators.java]: Will do, left them there for review's sake.

File: bootstrap/standalone/src/main/java/org/geysermc/platform/standalone/Util.java (hunk 2 above)
Comment (e3ndr) [reply bootstrap/standalone/src/main/java/org/geysermc/platform/standalone/Util.java]: Warning was used to standout in the console.

Comment (RednedEpic) [reply bootstrap/standalone/src/main/java/org/geysermc/platform/standalone/Util.java]: You can just use the ChatColor enum iirc and colors will automatically apply. Use a bright color from that instead.

**Pull request merged**